CODING_CONVENTIONS.md

```diff
@@ -259,23 +259,23 @@ An examplary doxygen documentation in a header file can look like this.
 
 ```
 /*
- * Copyright (C) 2014 Peter Schmerzl <peter@schmerzl-os.org>
+ * Copyright (C) 2014 your_name yourlastname <youremailadrr@mail.com>
  *
  * This file is subject to the terms and conditions of the GNU Lesser General
  * Public License v2.1. See the file LICENSE in the top level directory for more
  * details.
  */
```

Comment:
we are not using this license header

Reply:
I should use this?
*/ Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0

Reply:
Yep, I think it's better, in the same line as the proyect license,.
also may be a good Idea to put a reference for the original repo in the footer of this document, what do you think about it?